code/script.js

```diff
@@ -35,17 +42,52 @@ const showMessage = (message, sender) => {
 
 // Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
+  showMessage("Welcome to report error messages about our municipal streets! Do you want to make a report?", 'bot');
+};
+
+
+
+  
+  // Display the user's answer in the chat
+  // Here you can process the user's answer, ask the next question, etc.
+  // For now, let's just display a response from the bot:
+
+
+
+
+
+const handleUserAnswer = event => {
+  event.preventDefault ();
+  const userAnswer = userInput.value.trim().toLowerCase();
+  showMessage(userAnswer, 'user');
+  setTimeout(() => {
```

Comment:
After showing the message from the user, you can clear the value from the input field by writing `userInput.value = ''`.